src/components/panes/declared-variables-pane.jsx

```diff
@@ -12,28 +12,50 @@ import tasks from '../../actions/task-definitions'
 export class DeclaredVariablesPaneUnconnected extends React.Component {
   static propTypes = {
     userDefinedVarNames: PropTypes.arrayOf(PropTypes.string),
+    environmentVariables: PropTypes.arrayOf(PropTypes.string),
   }
 
   shouldComponentUpdate(nextProps) {
     return !deepEqual(this.props, nextProps)
   }
 
   render() {
+    const environmentalDVs = Object.keys(this.props.environmentVariables).length ?
+      Object.keys(this.props.environmentVariables).map(varName =>
+        (<DeclaredVariable
+          value={this.props.environmentVariables[varName][0]}
+          varName={varName}
+          key={varName}
+        />)) :
+      undefined
+    const edvElem = environmentalDVs !== undefined ? (
+      <div className="declared-variables-list">
+        <h3>Saved Environment</h3>
+        {environmentalDVs}
+      </div>
+    ) : undefined
```

Comment:
maybe simplify this into a single if/else block rather than repeating the ` ? : ` pattern twice?

Reply:
sure will do